packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -0,0 +1,36 @@
+package com.amazonaws.amplify.amplify_datastore
```

Comment:
Don't forget to add license boilerplate.

Reply:
Added.